src/components/AppModal.vue

```diff
@@ -0,0 +1,152 @@
+<template>
+  <div class="container" :class="{ inactive: !isModalActive }">
+    <div class="header">
+      <h2>{{ modalTitle }}</h2>
+      <i class="fa-solid fa-x" @click="toggleAppModal"></i>
+    </div>
+    <div class="form_body">
+      <slot />
+    </div>
+    <div class="footer">
+      <button class="cancel" @click="toggleAppModal">Cancel</button>
+      <button class="save">Save</button>
+    </div>
+  </div>
+</template>
+
+<script>
+import { mapState, mapMutations } from 'vuex';
+
+export default {
+  name: 'AppModal',
+  props: {
+    modalTitle: String,
+  },
+  computed: {
+    ...mapState({
+      isModalActive: 'isModalActive',
+    }),
+  },
+  methods: {
+    ...mapMutations({ toggleAppModal: 'toggleAppModal' }),
+  },
+};
+</script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style lang="scss" scoped>
+/* 375px Mobile */
```

Comment:
These breakpoint comments are not really necessary, the mixins are semantic enough
also improve these styles as we discussed previously, regarding `width` and `max-width`